lib/routes/lnurl/payment/lnurl_payment_handler.dart

```diff
@@ -63,7 +63,7 @@ Future<LNURLPageResult?> handlePayRequest(
         _log.info("LNURL payment success, action: ${result.data}");
         return LNURLPageResult(
           protocol: LnUrlProtocol.Pay,
-          successAction: result.data,
+          successAction: result.data.successAction,
         );
       } else if (result is LnUrlPayResult_EndpointError) {
```

Comment:
Do we need to handle `LnUrlPayResult_PayError` as well?

Reply:
No, from what I understand this will be null when an error is returned.

```                           return Ok(LnUrlPayResult::PayError {
                                data: LnUrlPayErrorData {
                                    payment_hash: invoice.payment_hash,
                                    reason: e.to_string(),
                                },
```

https://github.com/breez/breez-sdk/blob/95f5e040dfb8deb10efe3b54602b34aabbf22d6a/libs/sdk-core/src/breez_services.rs#L325-L329

Reply:
Sorry, I answered the wrong question. 

Agree with your latest comment with handling LnUrlPayResult_PayError too.